### PR TITLE
Handling environment deserialization panic with envy, adding futures to cargo :P 

### PR DESCRIPTION
**Pull request by Vi-Sri**
small and rather useless PR to get the contribution feels. LOL. 

Handled Panic during envy deserialization. We can write small macros for some handy stuff. Also its always good to use unwrap_or_else rather than unwrap :P

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -11,16 +12,21 @@ fn init_logger() {
...
 }
 
 fn main() {
     init_logger();
 
-    let config = config::get_config().unwrap();
+    let config = config::get_config().unwrap_or_else(
+        |e| {
+            eprintln!("Problem deserializing environment variables: {}", e);
+            process::exit(1);
```
Comment (AjithPanneerselvam) [inline src/main.rs:24]: How about replacing these 2 lines with log fatal @Vi-Sri?

Comment (Vi-Sri) [reply src/main.rs:24]: yeahh but unwrap_or_else is kinda cool.. 😅

**Comment by AjithPanneerselvam** [comment]
Lol, yes you are right. I was too lazy to handle Result/Option. Thanks for this PR 😄

File: src/main.rs (hunk 1 above)
Comment (AjithPanneerselvam) [reply src/main.rs:24]: Yeah `unwrap_or_else` can stay. We can replace line 23 and 24 by log fatal.

Comment (Vi-Sri) [reply src/main.rs:24]: @AjithPanneerselvam oh you talked about the replacing the eprintln and process exit. yeah. log fatal is definitely a better option inside the closure. lol i misunderstood.

**Pull request merged**